.github/workflows/ci_trie_benchmark.yml

```diff
@@ -35,3 +35,11 @@ jobs:
       - name: Install Rust
         if: ${{ steps.cache.outputs.cache-hit != 'true' }}
         uses: dtolnay/rust-toolchain@stable
+
+      - name: Benchmarks
+        uses: boa-dev/criterion-compare-action@v3
+        with:
+          cwd: "crates/common/trie"
+          benchName: "trie_bench"
+          branchName: ${{ github.base_ref }}
```

Comment:
I think instead of a branch name, it should be the commit. Because `base_ref` might be just `main` and you run into the same issue. But looking at the action it seems it does not support commits, bummer.

Reply:
Do you think we should go back to a more manual approach to use the base branch always? Or should I keep this action as is?

Reply:
It's fine as it, you just need to make sure you're in sync with main. I don't see it as a big issue